src/modules/localStorage.js

```diff
@@ -0,0 +1,13 @@
+export const loadDataLs = () => {
+  const data = localStorage.getItem('library');
+  if (data) {
+    const library = JSON.parse(data);
+    return library;
+  }
+  return [];
+};
```

Comment:
- [x] Please consider changing the logic from this function to a simpler one like:
`export const loadDataLs = () => localStorage.getItem('library') ? JSON.parse(data) : [ ]`
since it is the same and it helps you keep your code simple (KISS).

Reply:
Apparently, the one-line code doesn't function, because `data`  is not defined. I will try another approach.